packages/webapp/package.js

```diff
@@ -20,9 +20,9 @@ Npm.strip({
   useragent: ["test/"]
 });
 
+// whitelist plugin is now included in the core
```

Comment:
Nice comment about the change, but that should have been just a comment here, no need to have it in the code itself.